src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/Functions/ProcessInitialDataSetVersionFunction.cs

```diff
@@ -69,9 +67,9 @@ public class ProcessInitialDataSetVersionFunction(
         await UpdateImportStage(dataSetVersionImport, DataSetVersionImportStage.WritingDataFiles, cancellationToken);
         await parquetService.WriteDataFiles(dataSetVersionImport.DataSetVersionId, cancellationToken);
     }
-
-    [Function(ActivityNames.CompleteProcessing)]
-    public async Task CompleteProcessing(
+    
```

Comment:
Minor one, there's some whitespace crept in here into `ProcessInitialDataSetVersionFunction`.

Reply:
Heh - I'm not seeing what you're seeing I'm afraid.  I have a method `CompleteInitialDataSetVersionProcessing` which has no additional whitespace under it

Reply:
It's not a big deal but it's strange that you are not seeing this. The introduction of that line containing only whitespace is shown on Github when I look through the 'Files changed' list. It's line 70 directly above the line containing the annotation `[Function(ActivityNames.CompleteInitialDataSetVersionProcessing)]`.

If I check out this branch locally and reformat this file it shows as a change which can be seen here:

![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/4147126/3532f7fd-0807-4e1a-998b-3cb202ee945a)

Reply:
Aha ok got it ta

Reply:
fixed!